datadog_checks_dev/datadog_checks/dev/tooling/create.py

```diff
@@ -51,8 +51,15 @@ def construct_template_fields(integration_name, repo_choice, **kwargs):
         author = 'U.N. Owen'
         email = email_packages = 'friend@datadog.community'
         install_info = (
-            'The {} check is not included in the [Datadog Agent][2] package, so it must\n'
-            'be installed manually.'.format(integration_name)
+            'To install the {integration_name} check on your host:\n\n'
+            '    1. Install the [developer toolkit]'
```

Comment:
No need to indent numbers

Reply:
Missing link too

Reply:
Link is there, just too large to fit on one line and have style pass.